eventually/authentication/models.py

```diff
@@ -9,6 +9,7 @@
 
 from django.db import models
 from django.contrib.auth.base_user import AbstractBaseUser
+import django.db.utils
 
 
 class CustomUser(AbstractBaseUser):
```

Comment:
extend class docstring with sphinx's `param` and `type` directives for each attribute.

Reply:
Fixed